### PR TITLE
Updating PipeConfig to PipeConfiguration in example

### PR DESCRIPTION
**Pull request by secondsun**

### CODE REVIEW

**Comment by secondsun** [comment]
@danielpassos @qmx :eyeglasses: ?

File: docs/guides/aerogear-android/AerogearAndroidPagination.asciidoc [hunk 1]
```diff
@@ -55,17 +55,17 @@ void onCreate() {
      ResponseParser responseParser = new GsonResponseParser();
      responseParser.setMarshallingConfig(marshallingConfig);
 
-     PipeConfig pipeConfig = new PipeConfig(BASE_URL, Data.class);//Data is just an example type
-     pipeConfig.setResponseParser(responseParser);
+     RestfulPipeConfiguration config = PipeManager.config("listing", RestfulPipeConfiguration.class)
+                .withUrl(BASE_URL)
+                .responseParser(responseParser))
+                .pageConfig(pageConfig)
+                .forClass(Data.class);
```
Comment (danielpassos) [inline docs/guides/aerogear-android/AerogearAndroidPagination.asciidoc]: `forClass` return `Pipe<DATA>` no a RestfulPipeConfiguration

**Comment by danielpassos** [comment]
Need replace `/docs/specs/aerogear-android/` by `/docs/specs/aerogear-android-pipe/`

**Comment by secondsun** [comment]
@danielpassos "fixeD"  the javadoc links will work when we push the 2.0 docs.

**Comment by danielpassos** [comment]
:+1:

**Pull request merged**